leo_description/urdf/macros.xacro

```diff
@@ -5,8 +5,7 @@
                params="default_antenna:=true
                        rockers_fixed:=true
                        footprint_link:=true
-                       link_prefix:=''
-                       joint_prefix:=''
```

Comment:
Removing the `joint_prefix` macro argument completely breaks backward compatibility. I suggest leaving it as it may be used in some scenarios.

Reply:
Done